.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup
         uses: pnpm/action-setup@v4
         with:
-          version: 8
+          version: 8.15.6
```

Comment:
Using this version because it's currently the latest we support on StackBlitz.com.